src/nanoemoji/config.py

```diff
@@ -168,18 +169,39 @@ def output_format(self):
         return Path(self.output_file).suffix
 
     @property
-    def has_bitmaps(self):
-        return self.color_format.startswith("sbix") or self.color_format.startswith(
-            "cbdt"
+    def has_bitmaps(self) -> bool:
+        return bool({"sbix", "cbdt"}.intersection(self.color_format.split("_")))
+
+    @property
+    def has_picosvgs(self) -> bool:
+        return bool(
+            {"glyf", "colr", "picosvg", "picosvgz"}.intersection(
+                self.color_format.split("_")
+            )
+        )
+
+    @property
+    def has_untouchedsvgs(self) -> bool:
+        return bool(
+            {"untouchedsvg", "untouchedsvgz"}.intersection(self.color_format.split("_"))
         )
 
     @property
-    def has_picosvgs(self):
-        return not (self.color_format.startswith("untouchedsvg") or self.has_bitmaps)
+    def has_svgs(self) -> bool:
+        return self.has_picosvgs or self.has_untouchedsvgs
 
     @property
-    def has_svgs(self):
-        return not self.has_bitmaps
+    def is_vf(self) -> bool:
+        return len(self.masters) > 1
+
+    @property
+    def is_ot_svg(self) -> bool:
+        return bool(
+            set(
+                "".join(p)
+                for p in itertools.product(("picosvg", "untouchedsvg"), ("", "z"))
+            ).intersection(self.color_format.split("_"))
+        )
```

Comment:
isn't this the same as "svg" in color_format?

Reply:
well, right now yes.. but who knows, for whatever reason, there may appear another color_format which contains the substring "svg" but doesn't produce an OT-SVG table (which is what this `is_ot_svg` property is checking). I prefer to keep it that way.